### PR TITLE
Reduce traffic cache size.

### PR DESCRIPTION
**Pull request by darina**

### CODE REVIEW

File: drape_frontend/drape_engine.cpp [hunk 1]
```diff
@@ -554,7 +554,7 @@ void DrapeEngine::UpdateTraffic(traffic::TrafficInfo const & info)
 #endif
 
   df::TrafficSegmentsColoring segmentsColoring;
-  segmentsColoring.insert(make_pair(info.GetMwmId(), info.GetColoring()));
+  segmentsColoring.emplace(info.GetMwmId(), info.GetColoring());
```
Comment (ygorshenin) [inline drape_frontend/drape_engine.cpp]: Is it guaranteed that segmentsColoring doesn't have key equal to info.GetMwmId()?

Comment (darina) [inline drape_frontend/drape_engine.cpp]: он же пустой)

Comment (ygorshenin) [inline drape_frontend/drape_engine.cpp]: Please, at least add a comment that you aren't interested in info.GetColoring() if segmentsColoring already contains info for the mwm.

Comment (ygorshenin) [inline drape_frontend/drape_engine.cpp]: OK :)

**Comment by rokuz** [comment]
LGTM

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**